src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php

```diff
@@ -23,6 +23,22 @@ class MySqlGrammar extends Grammar
     protected $serials = ['bigInteger', 'integer', 'mediumInteger', 'smallInteger', 'tinyInteger'];
 
     /**
+     * Get the bindings for determining table information including database name.
+     *
+     * @param Connection $connection
+     * @param            $table
```

Comment:
please don't align these, and use exactly 2 spaces between param, type, and name